Golden_Path.postman_collection.json

```diff
@@ -13056,41 +12898,37 @@
 					"response": []
 				},
 				{
-					"name": "Check Send Transfer 4 Decimal #949",
+					"name": "Check Send Quote 4 Decimal #949",
 					"event": [
 						{
 							"listen": "test",
 							"script": {
-								"id": "900142b0-e4d7-43a4-a751-38202b600661",
+								"id": "4376c794-71a8-474c-9b25-a0714ca62df8",
 								"exec": [
 									"pm.test(\"Status code is 202\", function () {",
 									"    pm.response.to.have.status(202);",
 									"});",
 									"",
-									"//Check the request that Switch forwards to payeefsp",
+									"//Check Data on payer side",
 									"setTimeout(function () {",
-									"  pm.sendRequest(pm.environment.get(\"HOST_SIMULATOR\")+\"/payeefsp/requests/\"+pm.environment.get(\"transfer_ID\"), function (err, response) {",
-									"          if(response.responseSize !== 0) {",
-									"                ",
-									"                ",
-									"                //Check the data",
-									"                var jsonData = response.json().data;",
-									"",
-									"                pm.test(\"payeefsp data should have the same amount as request and 4 decimal checked\", function () {",
-									"                   pm.expect(jsonData.amount.amount).to.eql('1.9999');",
-									"                });",
-									"",
-									"                ",
-									"          } else {",
-									"              pm.test(\"Transfer FAILED\", function () {",
-									"                throw new Error('Did not receive response');",
-									"              });",
+									"  pm.sendRequest(pm.environment.get(\"HOST_SIMULATOR\")+\"/payerfsp/callbacks/\"+pm.variables.get(\"quoteId\"), function (err, response) {",
+									"      if(response.responseSize !== 0) {",
+									"            ",
+									"            var jsonData = response.json().data;",
+									"            ",
+									"            pm.test(\"Send Quote Amount in 4 decimal returned\", function () {",
+									"            pm.expect(jsonData.transferAmount.amount).equal(\"60.1234\");",
```

Comment:
Pls use the "amount" variable from the payload of the request in the assertions rather using a constant number.

Reply:
This will be dealt in #987(https://github.com/mojaloop/project/issues/987) and planned to have separate collection for bug fixes.

Reply:
Created a story to resolve this --> https://github.com/mojaloop/project/issues/1007